releases/v19.2.0-beta.20190826.md

```diff
@@ -84,6 +84,13 @@ $ docker pull cockroachdb/cockroach-unstable:v19.2.0-beta.20190826
 - CockroachDB now supports an optional frame exclusion clause in the specification of window frames. [#28262][#28262] {% comment %}doc{% endcomment %}
 - Enabled collection of histograms by default during automatic statistics collection. [#39743][#39743]
 - The `force_retry()` function, which previously required admin privileges, is now available to all users to test their transaction retry loops. [#39246][#39246] {% comment %}doc{% endcomment %}
+- When [`DROP CONSTRAINT`](../v19.2/drop-constraint) is executed in a [transaction](../v19.2/transaction.html) on a validated constraint, it no longer takes effect immediately, and subsequent writes in that constraint will now fail if they violate that constraint. This is to enforce consistency when dropping constraints asynchronously in the schema changer. [#39648][#39648]
```

Comment:
Should `subsequent writes in that constraint` be shorted to just `subsequent writes`..?